SpatialGDK/Extras/schema/rpc_components.schema

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+type UnrealRPCPayload {
+    uint32 offset = 1;
+    uint32 rpc_index = 2;
+    bytes rpc_payload = 3;
+}
+
+type Void {
+}
+
+component UnrealClientRPCEndpoint {
+    id = 9990;
+    event UnrealRPCPayload unreliable_client_to_server_rpc;
+    command Void reliable_server_to_client_rpc(UnrealRPCPayload);
```

Comment:
Void with a capital? I think I'm going to be sick.

Reply:
It makes me sad too